CHANGELOG.md

```diff
@@ -49,6 +49,7 @@
 
 <a name="breaking_changes_1.15.0">[Breaking Changes:](#breaking_changes_1.15.0)</a>
 
+- [monaco] upgraded to monaco 0.23.0 including replacement of `quickOpen` API (0.20.x) with `quickInput` API (0.23.x) [#9154](https://github.com/eclipse-theia/theia/pull/9154)
```

Comment:
As #9154 has been merged after the release of 1.15.0, I'd argue that this should be moved to the breaking changes section of 1.16.0.

Also `upgraded to monaco 0.23.0` will probably part of the normal changelog. Therefore, the breaking changes should only include `replacement of 'quickOpen' API (monaco 0.20.x) with 'quickInput' API (monaco 0.23.x)`

Reply:
Right! updated...